src/status_im/contexts/onboarding/enable_notifications/view.cljs

```diff
@@ -69,7 +75,10 @@
 
 (defn view
   []
-  (let [insets (safe-area/get-insets)]
+  (let [insets                (safe-area/get-insets)
+        {:keys [status-type]} (rf/sub [:multiaccount/current-user-visibility-status])]
+    (when (nil? status-type)
+      (dispatch-visibility-status-update constants/visibility-status-automatic))
```

Comment:
@ilmotta @flexsurfer Any idea why in this PR we are changing the `visibility-status`?

This behavior is not being moved from the welcome screen to this screen, I think it's a new behavior added. Since I don't understand what's doing, I'd suggest removing it from the PR, seems it's not part of the main issue :thinking:

Reply:
cc @Parveshdhull

Reply:
> I'd suggest removing it from the PR,

yes we should remove this.

> I don't understand what's doing

Initially we added this in welcome screen, and as we are removing welcome screen we are moving this call here to enable notification screen.

But even in welcome screen this was not needed and was removed in https://github.com/status-im/status-mobile/pull/21068

Reply:
Thanks for the explanation @Parveshdhull  I'll remove it :+1: